.github/.dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      # Framework gems
+      - dependency-name: jest
+        dependency-type: direct
```

Comment:
Should we add standard in here too? 

I got caught in a pondering of this yesterday when tinkering with adding the new scss linter and came to the conclusion we should https://github.com/alphagov/content-publisher/commit/1b8abf68f8be9874fc51826aa6795a7f3e8ef59a

Let me know if you have thoughts either way as it seemed a trade-off

Reply:
Yes, that seems reasonable - it's basically a stand-in for rubocop-govuk, (albeit not maintained by GDS), and we'd want to know if linting conventions change.
I left it out as it's not a thing we use to build our code 'on', so to speak - but will add it in now.

Reply:
Great, thank you :) Once this is merged I'll do some work to clean up these lingering ones.